sdk/core/azure-core-amqp/pom.xml

```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core;current} -->
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
```

Comment:
@srnagar Does this need a current version dependency on Azure Core?

Reply:
Hopefully it doesn't.